jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile

```diff
@@ -7,7 +7,7 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
-    base_versions: ['2019.1', '3.1'],
+    base_versions: ['2019.1', '3.1', '3.0', '2.9'],
```

Comment:
We don't have 2.9.
And we only support to upgrade from latest release, so for master & 3.2, we only need to test upgrade from 2019.1 and 3.1

Reply:
it was test. all removed